erpnext/stock/doctype/serial_no/serial_no.py

```diff
@@ -160,7 +160,7 @@ def before_rename(self, old, new, merge=False):
 	def after_rename(self, old, new, merge=False):
 		"""rename serial_no text fields"""
 		for dt in frappe.db.sql("""select parent from tabDocField
-			where fieldname='serial_no' and fieldtype='Text'"""):
+			where fieldname='serial_no' and fieldtype like '%%Text%%'"""):
```

Comment:
@akshay83, can you add the `fieldtype in ('Text', 'Small Text')` condition instead?